### ISSUE TITLE
Assign also Field Groups to Categories

### ISSUE THREAD

**Issue by coolcat-creations** [joomla-projects/custom-fields#152]
For now it´s possible to assign Fields to a specific Category (Article Fields) 
It would be useful to assign a whole Field Group to a Category.

Only thing to think about is that if the Field Group is assigned to a category, the Field itself can just be assigned to it´s children categories.

**Comment by laoneo** [comment]
I guess this will be rather hard to implement.

**Comment by brianpeat** [comment]
I know this is a bit old, but it's something I really liked in FieldsAttach. Basically every group is a tab, and you can limit the group (tab) to certain categories. The result is you can make a tab only show in the editor in articles in a specific category, which is really nice for end users.

It does look like you can get this behavior, but you have to make sure every field in the group is set to the specific category, so you could end up having to make lots of duplicate fields. I guess the result can be the same, just a different way of thinking. If you made the groups able to be limited by category and allowed fields to be in more than one group, it might feel more flexible.

**Comment by zero-24** [comment]
I think this should be moved to the cms repo as com_fields is merged into the core now?

**Comment by brianpeat** [comment]
Yeah, I'm still learning where everything is. At this point I doubt it'll be worth pursuing as you CAN get the same result since the tab doesn't appear if you limit all the fields in the group.
